en/Help and support.md

```diff
@@ -0,0 +1,34 @@
+Learn how you can get help and support resources when using Obsidian or any of our paid services.
+
+## Contact Obsidian support
+
+For questions and other support related to our paid services, such as [[Introduction to Obsidian Publish|Obsidian Publish]] and [[Introduction to Obsidian Sync|Obsidian Sync]], you can [email the Obsidian team](mailto:support@obsidian.md).
```

Comment:
Would be nice to make it clear that we also provide help for paid licenses (commercial, catalyst), as well as general account & billing issues. There's really no way for them to resolve them otherwise!